php/libraries/Database.class.inc

```diff
@@ -895,7 +895,7 @@ class Database
             $query = "SELECT * FROM $table WHERE $where";
 
             // select the database data
-            $this->select($query, $oldRows);
+            $oldRows = $this->pselect($query, array());
```

Comment:
Yes, it is required for the tear down of my helper function that deletes the subprojects.